src/libraries/System.Private.CoreLib/src/System/Collections/Generic/IDictionaryDebugView.cs

```diff
@@ -17,12 +17,17 @@ public IDictionaryDebugView(IDictionary<K, V> dictionary)
         }
 
         [DebuggerBrowsable(DebuggerBrowsableState.RootHidden)]
-        public KeyValuePair<K, V>[] Items
+        public DebugViewDictionaryItem<K, V>[] Items
```

Comment:
Any chance you could rename the type parameters to be `TKey` and `TValue` here as well?

Reply:
Done.